src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -43,6 +57,21 @@ public string Engine
             }
         }
 
+        /// <summary>
+        /// Return the display name of python engine enum.
+        /// </summary>
+        public string Engine
+        {
+            get
+            {
+                return engine.GetType()?
+                    .GetMember(engine.ToString())?
+                    .First()?
+                    .GetCustomAttribute<DisplayAttribute>()?
+                    .Name;
```

Comment:
`ToString` should actually work and it is much simpler. We could get rid of the attribute.

Reply:
@mmisol Thanks!